openfisca_core/holders.py

```diff
@@ -360,17 +360,27 @@ def put_in_cache(self, value, period, extra_params = None):
     def get_from_cache(self, period, extra_params = None):
         return self if self.column.is_permanent else DatedHolder(self, period, extra_params)
 
-    def get_extra_param_names(self):
-        return self.formula.function.__func__.func_code.co_varnames[3:]
+    def get_extra_param_names(self, period = None):
```

Comment:
Why set a default value of `None` for `period`? Why not simply force the parameter to exist? This would be more predictable.